packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -98,9 +100,22 @@ export class SDKClient {
   private operatorAccountGauge;
   private maxChunks;
   private fileAppendChunkSize;
+  private mirrorNodeClient;
```

Comment:
```suggestion
  private readonly mirrorNodeClient: MirrorNodeClient;
```

Reply:
Same for other lines above (98-102):
- please add types to those variables
- change them to `readonly` if we never reassign them anywhere in the class